scio-bigquery/src/it/scala/com/spotify/scio/bigquery/BigQueryClientIT.scala

```diff
@@ -85,7 +85,7 @@ class BigQueryClientIT extends AnyFlatSpec with Matchers {
   }
 
   it should "support missing source tables" in {
-    bq.query.extractLocation("SELECT 6") shouldBe None
+    bq.query.extractLocation("SELECT 6") shouldBe Some("EU")
```

Comment:
So this means the query will be executed in EU despite not having any source table.
I'm not sure how it defaults to EU though, a project-wide setting?

Reply:
this seems like it's probably reading your local config?